packages/modal/src/components/modal/Modal.module.css

```diff
@@ -6,6 +6,7 @@
   --swui-modal-width: 960px;
   --swui-modal-header-border-color: var(--lhds-color-ui-300);
   --swui-modal-shadow: var(--swui-shadow-modal);
+  --swui-sticky-modal-footer-shadow: -1px 0px 25px -2px rgba(0, 0, 0, 0.2);
```

Comment:
No custom shadows in the components. Use one of the shadows in default-theme. Or add a new one to default-theme.

Reply:
Fixed!